.vscode/settings.json

```diff
@@ -6,8 +6,5 @@
     "search.exclude": {
         "out": true // set this to false to include "out" folder in search results
     },
-    "typescript.tsdk": "./node_modules/typescript/lib", // we want to use the TS server from our node_modules folder to control its version
-    "files.trimTrailingWhitespace": true,
-    "clang-format.executable": "${workspaceFolder}/node_modules/.bin/clang-format",
```

Comment:
There is a reference to clang-format in the CONTRIBUTING.md in the root folder. We'll need to keep the "formatting" section consistent with the changes we're making.